test-support/pages/components/_component.js

```diff
@@ -41,39 +41,40 @@ export default function component (scope = "", descriptor = {}) {
     descriptor = scope
     scope = null
   }
-
-  return {
-    ...(scope ? { scope, itemScope : scope } : {}), // inject the scope only if it was provided
```

Comment:
So you ultimately got rid of the spread operator... The rest of the CR makes no sense to me. :confused:

Reply:
yeah, after being able to load the transform plugin the use of it wasnt possible for some unknown reason.  i figured for this reason alone you would reject the PR because the real solution is to ensure we can use the spread operator.